index.html

```diff
@@ -17,34 +17,34 @@ <h2>Mark</h2>
     </header> 
```

Comment:
- [ ] kindly as you fix the header as requested by the previous reviewer kindly consider sticking to the Figma design as much as possible by displaying all the nav items and giving the logo the same font size as given in the design to make your project an exact duplicate of the given design check the images below for more clarification 
- your design 👇 
<img width="503" alt="Screen Shot 2023-05-12 at 23 34 29" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/60146030/887398e0-4bb5-4ee4-a89a-1bdcfbef1f18">


- Figma design 👇 
<img width="467" alt="Screen Shot 2023-05-12 at 23 36 23" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/60146030/1ba317fa-ff1b-4d88-8804-49edf20bf5cc">

Reply:
Fixed the issue